pkg/fab/chconfig/chconfig.go

```diff
@@ -458,6 +458,8 @@ func loadConfigGroupPolicies(versionsGroup *common.ConfigGroup, group *common.Co
 
 func loadConfigPolicy(versionsPolicy *common.ConfigPolicy, configPolicy *common.ConfigPolicy) error {
 	versionsPolicy.Version = configPolicy.Version
+	versionsPolicy.Policy = configPolicy.Policy
+	versionsPolicy.ModPolicy = configPolicy.ModPolicy
```

Comment:
I'm fine with setting the two fields, but loadPolicy should be deleted since it doesn't do anything.

Reply:
done